api.ts

```diff
@@ -161,6 +161,8 @@ export interface SyncTableDef<
   description?: string;
   /** See {@link SyncTableOptions.schema} */
   schema: SchemaT;
+  /** See {@link SyncTableOptions.identityName} */
+  identityName: string;
```

Comment:
I wonder if there's a space to add a documentation indicating it's only used by dynamic sync tables.